allure-report-face/src/components/widgets-grid/WidgetsGridView.js

```diff
@@ -7,7 +7,9 @@ import {className} from '../../decorators';
 import allurePlugins from '../../pluginApi';
 
 const widgetTpl = (id) => `<div class="widget island" data-id="${id}">
-    <div class="widget__handle fa fa-arrows"></div>
+    <div class="widget__handle">
+        <span class="fa fa-ellipsis-v"></span> <span class="fa fa-ellipsis-v"></span>
```

Comment:
svg is not plain css :)
`background-image: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='5' height='5'><rect x='0' y='0' width='3' height='3' fill='%23ccc' /></svg>");`